website/cue/reference/components/transforms/pipelines.cue

```diff
@@ -74,22 +74,36 @@ components: transforms: pipelines: {
 			type: object: options: {
 				name: {
 					description: "Name of the pipeline"
-					warnings: []
-					required: false
-					common:   true
+					required:    false
+					common:      true
+					type: string: default: null
+				}
+
+				filter: {
+					description: """
+						A condition to filter the events that will be processed by the pipeline. If the conditions is not satisfied,
+						the event will be forwarded to the next pipeline.
+
+						The filter uses the same format that conditions use for [unit testing](\(urls.vector_unit_tests)).
```

Comment:
I might use the `filter` transform as the example here for conditions since that is closer to its use in the pipeline definition.